lib/stages/basebandReadout.cpp

```diff
@@ -47,7 +48,7 @@ basebandReadout::basebandReadout(Config& config, const string& unique_name,
     oldest_frame(-1),
     frame_locks(_num_frames_buffer),
     // Over allocate so we can align the memory.
-    baseband_data(std::make_unique<uint8_t[]>(_num_elements * _max_dump_samples + 15)),
+    data_buffer(_num_elements * _max_dump_samples + 15),
```

Comment:
This should be `2 * (_num_elements * _max_dump_samples + 15)` right? Because we can't guarantee that a contiguous piece of memory bigger than the length / 2 will be available (even with no writes queued).

Reply:
Agreed. I changed it and pushed the updated version.